### DISCUSSION TITLE
Failed to Fetch release from Cloudsmith

### DISCUSSION THREAD

**Discussion by stefanpinterBE** [rabbitmq/rabbitmq-server#9557] [Other] [answered]
### Describe the bug

hi!

when trying to build a rabbitmq image with Kolla (Openstack) I get this error on the building node:

E: Failed to fetch https://dl.cloudsmith.io/public/rabbitmq/rabbitmq-server/deb/ubuntu/dists/jammy/InRelease  402  Payment Required [IP: 13.32.110.64 443]

### Reproduction steps

1. run curl -v https://dl.cloudsmith.io/public/rabbitmq/rabbitmq-server/deb/ubuntu/dists/jammy/InRelease
2. see HTTP error 402
3. cry

### Expected behavior

i expected the download to work

**Comment by michaelklishin** [comment, marked as answer]
See https://github.com/rabbitmq/rabbitmq-server/discussions/9526
